draft-ietf-mls-architecture.md

```diff
@@ -440,23 +441,29 @@ latter delivery pattern via unicast channels (sometimes known as "client
 fanout"), broadcast channels ("server fanout"), or a mix of both.
 
 For the most part, MLS does not require the Delivery Service to deliver messages
-in any particular order. The one exception is that, because an MLS group has a
-linear history, the members of the group must agree on the order in which
+in any particular order. Applications can set policies that control their
+tolerance for out-of-order messages (see {{operational-requirements}}), and
+messages that arrive significantly out-of-order will be dropped without
+otherwise affecting the protocol. There are two exceptions to this. First,
+Proposal messages must all arrive before the Commit that references them.
+Second, because an MLS group has a linear history of epochs, the members of the
+group must agree on the order in which
 changes are applied.  Concretely, the group must agree on a single MLS Commit
 message that ends each epoch and begins the next one.
 
 In practice, there's a realistic risk of two members generating Commit messages
 at the same time, based on the same counter, and both attempting to send them to
 the group at the same time. The extent to which this is a problem, and the
 appropriate solution, depends on the design of the Delivery Service. Per the CAP
-theorem, there are two general classes of distributed system:
+theorem, there are two general classes of distributed system that the Delivery
```

Comment:
Minor:
- please add reference for CAP theorem
- a distributed DS in this context could be the clients and a single server, multiple (ex: federated servers), or clients and a network of servers. Probably worth mentioning that somewhere.